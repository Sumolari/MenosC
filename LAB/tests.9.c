{
	int numero[ 7];
	numero = -20;
	numero = numero++;
	numero = +20;
	numero = numero--;
	numero = 5 / 7;
	numero = 5 * 7;
	numero = 5 + 7;
	numero = 5 - 7;
}