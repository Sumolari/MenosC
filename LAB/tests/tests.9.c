{
	int numero;
	numero = -20;
	numero = numero++;
	numero = +20;
	numero = numero--;
	numero = 5 / 7;
	numero = 5 * 7;
	numero = 5 + 7;
	numero = 5 - 7;

	bool numero;

	bool bol;
	bol = -20;
	bol = bol++;
	bol = +20;
	bol = bol--;
	bol = 5 / 7;
	bol = 5 * 7;
	bol = 5 + 7;
	bol = 5 - 7;
	bol = true;
	bol = false;
	bol = 5 == 0 || 5 + 1 >= 7 && true;
}