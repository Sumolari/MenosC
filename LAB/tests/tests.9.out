Linea 12: Redeclaracion de variable
Linea 15: Tipo incompatible
Linea 16: Tipo incompatible
Linea 17: Tipo incompatible
Linea 18: Tipo incompatible
Linea 19: Tipo incompatible
Linea 20: Tipo incompatible
Linea 21: Tipo incompatible
Linea 22: Tipo incompatible

Numero de errores:      9
